server/routers/teams.js

```diff
@@ -0,0 +1,54 @@
+const express = require('express')
+const router = express.Router();
+const Team = require('../entities/database/context').Team
```

Comment:
Could we stick to using `context.<TableName>`, rather than doing it this way? I think it makes it more clear you are working with the database by doing that. What do you think @b0tw, @Yossarion?

Reply:
I agree!